src/WebJobs.Script/Config/ExtensionRequirementOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.ExtensionRequirements;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    public class ExtensionRequirementOptions
+    {
+        /// <summary>
+        /// Gets or Sets the minimum bundles configuration required for the function app.
```

Comment:
nit: only first letter is capitalized (applies to both of the summary comments in this file)

```suggestion
        /// Gets or sets the minimum bundles configuration required for the function app.
```